curp/src/server/cmd_worker.rs

```diff
@@ -380,9 +380,17 @@ mod tests {
         let ce = TestCE::new("S1".to_owned(), er_tx, as_tx);
         let cmd_board = Arc::new(RwLock::new(CommandBoard::new()));
         let spec_pool = Arc::new(Mutex::new(SpeculativePool::new()));
-        let exe_tx = start_cmd_workers(ce, spec_pool, cmd_board, Arc::new(Event::new()));
+        let exe_tx = start_cmd_workers(
+            ce,
+            spec_pool,
+            Arc::clone(&cmd_board),
+            Arc::new(Event::new()),
+        );
 
         let cmd = Arc::new(TestCommand::default());
+        {
```

Comment:
why do we wrap this line in a bracket? If we want to drop something for one-line code, just call `drop`.

Reply:
Historical legacies. Fixed now